### PR TITLE
Add .slugignore

### PR DESCRIPTION
**Pull request by citizen428**
## What type of PR is this? (check all applicable)

- [X] Optimization

## Description

This PR adds a [`.slugignore`](https://devcenter.heroku.com/articles/slug-compiler#ignoring-files-with-slugignore) file in order to reduce our slug size on Heroku which should speed up app start.

## Related Tickets & Documents

https://github.com/thepracticaldev/tech-private/issues/409

## Added tests?

- [X] no, because they aren't needed

## Added to documentation?

- [X] no documentation needed

### CODE REVIEW

**Comment by citizen428** [comment]
Feel free to suggest more things we can add to the file.

File: .slugignore [hunk 1]
```diff
@@ -0,0 +1 @@
+spec/
```
Comment (maestromac) [inline .slugignore]: I can think of  `scripts/` but unsure how big of a difference it would make

Comment (rhymes) [reply .slugignore]:
```shell
devto git:(master) du -chs spec scripts
8.6M	spec
8.0K	scripts
8.6M	total
```

:D

Comment (citizen428) [reply .slugignore]: Nothing here is huge :-/ Tbh the whole thing discussed in the issue probably gets resolved by running `git gc` on the slug.

**Pull request merged**